pkg/install/util.go

```diff
@@ -40,3 +42,39 @@ func ObjErrorf(obj kates.Object, format string, args ...interface{}) error {
 		obj.GetObjectKind().GroupVersionKind().Kind, obj.GetName(), obj.GetNamespace(),
 		fmt.Errorf(format, args...))
 }
+
+// AlreadyUndone means that an install action has already been undone, perhaps by manual user action
+type AlreadyUndone struct {
+	err error
+	msg string
+}
+
+func (e *AlreadyUndone) Error() string {
+	return fmt.Sprintf("%s: %v", e.msg, e.err)
+}
+
+func (e *AlreadyUndone) Unwrap() error {
+	return e.err
+}
+
+func NewAlreadyUndone(err error, msg string) error {
+	return &AlreadyUndone{err, msg}
+}
+
+// IsAlreadyUndone returns whether the given error -- possibly a multierror -- indicates that all actions have been undone.
```

Comment:
My high school journalism teacher would have yelled at you for putting spaces round the `--` em-dashes :P